README.md

```diff
@@ -1,8 +1,61 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Python, Statistical Analysis, R, OpenCL, Data Analysis
+
+**Familiar:** Virtulization, Testing Procedures, Data Modelling, Exploratory Data Analysis, Data Visulizations
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/linux_sql)]: Developed a Linux Cluster Monitoring system(LCM). In this project, the agent can be used to record hardware specifications of several nodes/servers which are connected with each other internally through a switch and communicate with IPv4. LCM is mainly used by Cluster Administration Team which manages and plans the resources for servers in a company. The data is then stored into a RDBMS server which can be used by the team to generate more information. This can help the administration team monitor and support the cluster in a better way.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/core_java)]:
+      
+  - Twitter App: In Progress
+  - JDBC App: In Progress
+  - Grep App: In Progress
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/springboot)]: In Progress
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/python_data_anlytics)]: In Progress
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/hadoop)]: In Progress
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/spark)]: In Progress
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/cloud_devops)]: In Progress
+
+
+## Highlighted Projects
+**MRI-Brain-Tumor-Segmentation** [[GitHub](https://github.com/KhanjanDabhi/MRI-Brain-Tumor-Segmentation)]: Developed a MRI brain tumor segmentation algorithm which would detect tumors from 3D MRI Images as a part of BRATS2020 Competition. Used Machine Learning for this project. The main goal of this project was to use of machine learning models outside a notebook enviorment while taking part in BRATS2020 challenge to detect tumor region in MRI images. Objectives learned include manipulation of 3D image data, using heavy and task specific deep neural networks such as UNet.
```

Comment:
**MRI-Brain-Tumor-Segmentation** [[GitHub](https://github.com/KhanjanDabhi/MRI-Brain-Tumor-Segmentation)]: Developed an MRI brain tumor segmentation algorithm which would detect tumors from 3D MRI Images as a part of BRATS2020 Competition. Used Machine Learning for this project. This project's main goal was to use machine learning models outside a notebook environment while taking part in BRATS2020 challenge to detect tumor region in MRI images. Objectives learned include manipulation of 3D image data, using heavy and task specific deep neural networks such as UNet.

more tech keyworkss and be specific